field.go

```diff
@@ -184,11 +184,19 @@ func Time(key string, val time.Time) zapcore.Field {
 // expensive (relatively speaking); this function both makes an allocation and
 // takes about two microseconds.
 func Stack(key string) zapcore.Field {
+	return String(key, takeStacktrace(1))
+}
+
+// StackSkip returns a stacktrace field like Stack and also allows for skipping
+// stack frames via the skip argument.
+//
+// StackSkip("foo",0) is equivalent to Stack("foo").
+func StackSkip(key string, skip int) zapcore.Field {
```

Comment:
I'm not sure if we should expose this right now -- I'd prefer to keep just `Stack` for now and think about whether we really need the skip version separately.